mobile/cmp/navigator/NavigatorModel.js

```diff
@@ -28,12 +28,21 @@ export class NavigatorModel extends HoistModel {
     /** @member {boolean} */
     track;
 
+    /** @member {boolean} */
+    enableSwipe;
+
     /** @member {RenderMode} */
     renderMode;
 
     /** @member {RefreshMode} */
     refreshMode;
 
+    /** @member {boolean} */
```

Comment:
@package is a new jsdoc annotation  ATM has been recommending -- could consider here.
Or no jsdoc, and gather them in a more clearly defined implementation block.